dfc.providerportal.findanapprenticeship/src/dfc.providerportal.findanapprenticeship/Services/ApprenticeshipService.cs

```diff
@@ -173,7 +174,7 @@ public IEnumerable<IDasProvider> ApprenticeshipsToDasProviders(List<Apprenticesh
                 evt.Metrics.TryAdd("Export failures", failure);
                 _telemetryClient.TrackEvent(evt);
 
-                return export;
+                return export.OrderBy(p => p.UKPRN).ToList();
```

Comment:
I've slipped this in to ensure that the results returned are always consistent, which will make future debugging easier.